docs/archive-data.md

```diff
@@ -0,0 +1,314 @@
+---
+id: archive-data
+title: How to archive data
+sidebar_label: Archive data
+---
+
+Data that is generated from a Workflow, such as Event Histories,  persists in normal Temporal storage per the configured retention period. The retention period exists so that the persistence store is not overwhelmed by the accummulation of Workflow data over time. To preserve Workflow data indefinitely, you can use the Archival feature which automatically moves the data from the normal persistence storage to long-term storage after the Workflow retention period expires.
+
+:::info
+
+**Archival is currently disabled by default.**
+- Archival is disabled when running Temporal via `docker-compose` and can not be enabled.
+- Archival is disabled by default when cloning the repo and installing the system manually, but can be enabled in the config.
+- Archival is disabled by default when deploying via [helm charts](https://github.com/temporalio/helm-charts/blob/d08ab3a582e9bc605da460b79cd7524e18d2c91d/templates/server-configmap.yaml#L176) but can be enabled in the config.
```

Comment:
Btw if you do a link like this it will lock the link to the specific commit hash when you created the link. Compare to this

https://github.com/temporalio/helm-charts/blob/master/templates/server-configmap.yaml#L176

which dynamically links to master.

Reply:
Just something to have in mind